question28/js/password.js

```diff
@@ -40,8 +42,21 @@ passwordToggleButtons.forEach((passwordToggleButton) => {
   passwordToggleButton.addEventListener('click', togglePassword); 
 });
 
+const setNewPassword = () => {
+  const newPasswordValue = passwordInputArea.value;
+  const registeredUserData = localStorage.getItem('userData'); 
+  const registeredUserJsonData = JSON.parse(registeredUserData);
+  registeredUserJsonData.password = newPasswordValue; 
+  const newUserData = JSON.stringify(registeredUserJsonData);
+  localStorage.setItem('userData', newUserData);
+}
+
 const init = () => {
-  console.log('未実装です');
+  setNewPassword ();
+  const takenForSetNewPassword = chance.apple_token();
```

Comment:
MUST
It was pointed out to me earlier that apple_token() is not appropriate, so I'm sharing!
It is not relevant to the context, so other methods must be used.
https://github.com/nakagawamai/js-lesson/pull/31#discussion_r1096366020
```suggestion
  const takenForSetNewPassword = chance.grid();
```

Reply:
Thank you for sharing about it! I learned a lot! Changed the use of guid() for the part where token is generated. Please confirm.

Fixed commit : https://github.com/csakyo/homework/pull/45/commits/3c7a7c46648cac398f218673003436eb5b614cd0